client/src/modules/ipr_tax/templates/impot_annuel.tmpl.html

```diff
@@ -0,0 +1,3 @@
+<div class="ui-grid-cell-contents text-right">
+  {{ row.entity.impot_annuel | currency: row.entity.currency_id }}
```

Comment:
All of these cell templates will take a long time to render.  It's best to define this using the column definitions for better performance.  This would be:
```js
{
  field: 'impot_annuel',
  cellClass : 'text-right',
  cellFilter : 'currency:row.entity.currency_id',
}
```

Try this for all the rest of the cells as well.